api_docs/swagger/swagger.yaml

```diff
@@ -0,0 +1,91 @@
+openapi: 3.0.0
+info:
+  title: OEM Partner API
+  version: 1
+  description: >-
+    API for creating and listing OEM Partners orders
+
+    ### Authorization header
+
+    The structure of this header is:
+
+
+    `ENCRYPTION_ALGORITHM COMPANY_ID:HMAC_SIGNATURE`
+
+
+    Each element is described in detail below
+
+    1. `ENCRYPTION_ALGORITHM`: tells the server which algorithm to use.
+       The only accepted value is `APIAuth-HMAC-SHA256`, other algorithms are
+       not accepted.
+
+    2. `COMPANY_ID`: your company's ID provided by SUSE. You can consider it as
+       a login.
+
+    3. `HMAC_SIGNATURE`: Base64-encoded SHA256-encrypted HMAC signature of the
+       following strings, joined by commas:
+
+       * HTTP verb
+
+       * `Content-Type` header
+
+       * `Content-MD5` header
+
+       * URL path
+
+       * `Date` header
+
+       E.g., `POST,application/json,xdcO0PyZn5SIAsIAI2pl7A==,/api/oem/partner_orders,Tue, 06 Jul 2016 04:39:43 GMT`
+
+       This string is then encrypted by the SHA256 HMAC using the key provided to you with the ID.
+
+    Given the example string above and an hypothetical company's ID `112233`,
```

Comment:
I think it's useful to link to the sample Bash script that can be used for testing https://github.com/SUSE/connect/blob/master/doc/OEM-API-Bash-Client.sh

Reply:
Done.